scripts/web.xml

```diff
@@ -23,7 +23,7 @@
         <servlet-class>org.glassfish.jersey.servlet.ServletContainer</servlet-class>
         <init-param>
             <param-name>javax.ws.rs.Application</param-name>
-            <param-value>io.swagger.inflector.SwaggerInflector</param-value>
+            <param-value>io.swagger.oas.inflector.SwaggerInflector</param-value>
```

Comment:
this class does not exist on inflector new version, you should use: `io.swagger.oas.inflector.OpenAPIInflector`